drizzlepac/outputimage.py

```diff
@@ -744,20 +766,25 @@ def deleteDistortionKeywords(hdr):
     """ Delete distortion related keywords from output drizzle science header
         since the drizzled image should have no remaining distortion.
     """
-    dist_kws = ['D2IMERR1','D2IMERR2','D2IMDIS1','D2IMDIS2','D2IM1.*','D2IM2.*',
-                'D2IMEXT', 'DP1', 'DP2', 'CPDIS?', 'CPERR?']
+    dist_kws = ['D2IMERR1', 'D2IMERR2', 'D2IMDIS1', 'D2IMDIS2',
+                'D2IMEXT', 'DP1', 'DP2', ]
+    dist_multi_kws = ['D2IM1.*', 'D2IM2.*', 'CPDIS?', 'CPERR?']
```

Comment:
Would it make sense to make this a global constant, especially if `addWCSKeywords()` is called often?

Reply:
This does make sense.  I will see about making these global constants...

Reply:
Done in 4b565a7